### PR TITLE
AB#105022-Display-logged-in-user

### PR DESCRIPTION
**Pull request by tropical-turtle**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Startup.cs [hunk 1]
```diff
@@ -77,8 +75,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IValidator<ApplicantSearchCriteria>, SearchValidator>();
 
             services.AddTransient<IMtoaArtifactService, MtoaArtifactService>();
-            //services.AddTransient<IUserGraphApiService, UserGraphApiService>();
-            services.AddTransient<IUserGraphApiService, MockUserGraphApi>();
+            services.AddTransient<IUserGraphApiService, UserGraphApiService>();
+            //services.AddTransient<IUserGraphApiService, MockUserGraphApi>();
```
Comment (AlexandreSanscartier) [inline src/CSF.SRDashboard.Client/Startup.cs:79]: Remove commented code.

**Review by AlexandreSanscartier — COMMENTED** [review]
Looks good!

File: src/CSF.SRDashboard.Client/Startup.cs (hunk 1 above)
Comment (TCPrimedPaul) [reply src/CSF.SRDashboard.Client/Startup.cs]: done

**Review by AlexandreSanscartier — APPROVED** [review]
Nice work.

**Comment by tropical-turtle** [comment]
Tested all look good.

**Pull request merged**